### DISCUSSION TITLE
No breadcrumb for returning to containing org

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4941] [Projects and Issues]
The project is within an org but there's no breadcrumb. That's also problematic for keeping track of which context someone is in.

![image](https://user-images.githubusercontent.com/2047520/128427949-cdf6c171-d3dc-448f-bfd8-51ba02984922.png)

**Comment by azenMatt** [comment]
Thanks @jdshkolnik - we are working on some UI updates here that will be live in the next couple of weeks that will help here.
